spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1574,24 +1574,37 @@ def append(self, text):
     @Slot()
     def paste(self):
         """
+        Insert text or file/folder path copied from clipboard.
+
         Reimplement QPlainTextEdit's method to fix the following issue:
         on Windows, pasted text has only 'LF' EOL chars even if the original
-        text has 'CRLF' EOL chars
+        text has 'CRLF' EOL chars.

```

Comment:
Remove this line.

Reply:
I did not understand what exactly to remove! Only this last line or whole the sentence, this actually was here before I made any change (This/previous PR)!

Reply:
Sorry, I think I got confused here. This line is fine, no need to change it.